runtime/src/chpl-cache.c

```diff
@@ -2839,7 +2839,7 @@ void chpl_cache_comm_prefetch(c_nodeid_t node, void* raddr,
   struct rdcache_s* cache = tls_cache_remote_data();
   chpl_cache_taskPrvData_t* task_local = task_private_cache_data();
   TRACE_PRINT(("%d: in chpl_cache_comm_prefetch\n", chpl_nodeID));
-  chpl_comm_diags_verbose_rdma("prefetch", node, size, ln, fn);
+  chpl_comm_diags_verbose_rdma("prefetch", node, size, ln, fn, 0);
```

Comment:
Is `0` a legal `commid`, or can that not happen?  If it can't happen then this is okay; otherwise we should arrange to print these without a `commid` because the `0` will be misleading.

Reply:
I think 0 is legal. I would be fine passing `-1` here or updating chpl_cache_comm_prefetch to receive a commID

Reply:
The latter (prefetch takes a `commid`) seems definitely preferable to me.

Reply:
I agree, but if it's a lot of work, I wouldn't hold up this PR for that.

Reply:
I will look at adding a commid argument to the prefetch call.